### PR TITLE
DOC Remove `filterwarnings` from `03_datetime_encoder.py`

### PR DESCRIPTION
**Pull request by Tialo**
Towards #766

Also there is generated file `doc/sg_execution_times.rst` which isn't in .gitignore, if that makes sense to add it there I can do it here or in separate PR.

### CODE REVIEW

**Review by Vincent-Maladiere — COMMENTED** [review]
Thanks, @Tialo. I have a small suggestion regarding `pprint`.

File: examples/03_datetime_encoder.py
Comment (Tialo) [reply examples/03_datetime_encoder.py:86]: Without pprint:
image
With print instead of pprint:
image


I think second option is better because all text can be seen on the screen

Comment (Tialo) [reply examples/03_datetime_encoder.py:86]: WDYT?

Comment (Vincent-Maladiere) [reply examples/03_datetime_encoder.py:86]: Ok, this makes sense. Let's replace `pprint` with `print` then.

Comment (Vincent-Maladiere) [reply examples/03_datetime_encoder.py:86]: After reviewing the notebook, `print(*..., sep="\n")` feels too complex. This is not what users will use.
Let's keep the `pprint` instead and add a `pprint` where you initially added a `print`.

Comment (Tialo) [reply examples/03_datetime_encoder.py]: Done!

**Comment by Tialo** [comment]
Also, what do you think about adding `doc/sg_execution_times.rst` in `.gitignore` since other generated files are included?

**Comment by Vincent-Maladiere** [comment]
> Also, what do you think about adding `doc/sg_execution_times.rst` in `.gitignore` since other generated files are included?

Yes, it sounds like a file to add to `.gitignore`. You can create an issue for this with a PR.

**Review by Vincent-Maladiere — APPROVED** [review]
LGTM, let's wait for the CI to be green before merging.

**Pull request merged**
